src/aap_eda/wsapi/consumers.py

```diff
@@ -208,9 +218,12 @@ def insert_event_related_data(self, message: AnsibleEventMessage) -> None:
 
     @database_sync_to_async
     def insert_audit_rule_data(self, message: ActionMessage) -> None:
-        job_id = message.job_id
-        job_instance = models.JobInstance.objects.filter(uuid=job_id).first()
-        job_instance_id = job_instance.id if job_instance else None
+        job_instance_id = None
+        if message.job_id:
```

Comment:
@hsong-rh I think this should be in a separate PR. I think the newer versions of pydantic does the uuid validation I am not sure if the version we are using is doing the validation.